src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
+  all = 'all',
```

Comment:
keys in enum should be uppercased (best practice)
```suggestion
  All = 'all',
```